x-pack/plugins/aiops/server/routes/log_rate_analysis/queries/fetch_categories.ts

```diff
@@ -76,7 +76,10 @@ export const getCategoryRequest = (
     timeFieldName,
     undefined,
     query,
-    wrap
+    wrap,
+    undefined,
+    undefined,
+    true
```

Comment:
Shouldn't this be `false`?

Reply:
Thanks for catching this, I got confused with the naming, fixed in [a666bc3](https://github.com/elastic/kibana/pull/176587/commits/a666bc3157128c12fdc3c98953781e86ee64860b).